app/services/malware_scanner.rb

```diff
@@ -17,16 +19,25 @@ def call
     MalwareScanResult.new(
       uploader: uploader,
       virus_found: virus_found?,
-      scan_result: scan_result,
+      scan_result: scan_result.stdout.strip,
```

Comment:
I think the strip should be used within the object constructor rather than on its use. So here:
```ruby
scan_result: scan_result.stdout
```
And below:
```ruby
OpenStruct.new(
  stdout: stdout.strip,
  status: status
)